### PR TITLE
style: Remove trailing whitespace

### PR DESCRIPTION
**Pull request by sijis**
Just a little tidying up.

### CODE REVIEW

**Review by vutny — COMMENTED** [review]
Good stuff. It is always annoying, when editing or changing doc/YAML layout, to handle all those completely unnecessary spaces.
Although, the changes become out of sync with `master` by now.

**Comment by sijis** [comment]
Rebased the PR.

**Comment by sijis** [comment]
Rebased again.

**Comment by sijis** [comment]
@waynesun09 if you could review this PR that would also be appreciated.

File: reportportal/README.md [hunk 1]
```diff
@@ -53,20 +53,15 @@ It describes installation of all mandatory services to run the application, and
...
+#### Requirements:
+- `RabbitMQ` (Helm chart installation)
+- `ElasticSearch` (Helm chart installation | Amazon Elasticsearch Service)
+- `PostgreSQL` (Helm chart installation | Amazon PostgreSQL RDS | Azure Database for PostgreSQL)
+- `MinIO` (Helm chart installation)
 
-All configuration variables are presented in `value.yaml` file.  
-
-Before you deploy ReportPortal you should have installed all its dependencies (requirements). Run Amazon RDS PostgreSQL or Azure Database for PostgreSQL, Amazon ES cluster in case you go with an external cloud services option.  
-
```
Comment (HardNorth) [inline reportportal/README.md:68]: Why do you remove these lines? Or it's just outdated branch?

**Review by HardNorth — CHANGES_REQUESTED** [review]

**Comment by HardNorth** [comment]
@sijis Last time rebase please.

**Pull request merged**